products/pnpm.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Pnpm
+category: app
+tags: package-manager javascript-runtime
```

Comment:
```suggestion
tags: build-tool javascript-runtime
```

See https://github.com/endoflife-date/endoflife.date/pull/4263#issuecomment-1855112609.

Reply:
that is not true. I am happy to remove it and add `package-manager` later, but I dont think adding build-tool is right move.

Reply:
`package-manager` not being an existing tag, it will have to be added in a separate PR (see documentation on https://endoflife.date/contribute). So you can file a separate issue for that if you want.